lua/efmls-configs/linters/codespell.lua

```diff
@@ -0,0 +1,19 @@
+-- Metadata
+-- languages: misc
+-- url: https://github.com/codespell-project/codespell
+
+local fs = require("efmls-configs.fs")
+
+local linter = "codespell"
+
+-- HACK piping to `sed` to change severity to warning
+local command = string.format("%s ${INPUT} | sed 's/^/w /'", fs.executable(linter))
```

Comment:
> (The codespell one uses a minor hack to change the severity level, since I couldn't figure out how to change the default severity. I can remove it if it's a problem.)

It is fine to leave it as default severity if there is no other way to determine what kind of diagnostic message it is. You can remove `sed` here as it will become platform-dependent.

This can become `%s --disable-colors ${INPUT}`

Reply:
Apparently, diagnostics without an explicit severity level get categorized as errors. Which is kinda inadequate for something like codespell, which should at most be warnings. Is there a way to change the default severity?

Reply:
(Otherwise removed the `sed` hack)